build.js

```diff
@@ -3,6 +3,13 @@
 const fs = require('fs');
 const https = require('https');
 const archiver = require('archiver');
```

Comment:
Add a newline here after the imports and before the definition of the global constants